ocean_grid_generator.py

```diff
@@ -992,7 +992,9 @@ def main(argv):
     r_dp = 0.0  # r value   of the displaced pole
     lat_dp = -99.0  # latitude of the displaced pole if input as arg
     lon_dp = 80.0  # longitude of the displaced pole
-    doughnut = 0.28 * 7 / 4 # fraction of dp grid to be excluded/cut, this particular value was used for the OM4 1/4 degree grid 
+    doughnut = (
```

Comment:
No thank you. I like the original better. I want "grep" to show what doughnut is set to.

Reply:
if you make the line shorter, black should leave it alone. This can be fixed later.

Reply:
the PEP8 coding style is very strict and it sometimes produces "weird" results. Most often it forces the developer to improve the readibility, e.g. 

```python
cond = np.where(phi_M < phi_enh_d)
# The last index with phi_M<phi_enh_d
j_c0d = cond[0][-1] 
```